pytest_mpl/plugin.py

```diff
@@ -196,6 +204,12 @@ def pytest_configure(config):
         if results_dir is not None:
             results_dir = os.path.abspath(results_dir)
 
+        default_style = (config.getoption("--mpl-default-style") or
+                         config.getini("mpl-default-style"))
```

Comment:
The default would need to be moved from the decorator to here:
```suggestion
                         config.getini("mpl-default-style") or
                         "classic")
```